provision/minikube/keycloak/config/kcb-infinispan-cache-config.xml

```diff
@@ -37,19 +37,23 @@
             </encoding>
             <memory max-count="10000"/>
         </local-cache>
-        <distributed-cache name="sessions" owners="2">
```

Comment:
The changes about owners and max-count should be active only when persistent sessions are enabled. At the moment Keycloak does those settings automatically in the `CacheManagerFactory`. Although this would prints an INFO message, I'd rather have the message in the log than maintaining two different configurations. So I'd ask you to revert the changes in this file.